src/components/collection/CollectionSkeletons.tsx

```diff
@@ -46,3 +46,23 @@ export function BindingsEditorSkeleton() {
         </Box>
     );
 }
+
+export function BindingsEditorSchemaSkeleton() {
```

Comment:
An attempt to create a skeleton that resembles a single JSON object.

Reply:
We probably need to make a skeleton folder soon as we start using them more often.